src/TreeHouse/BaseApiBundle/Behat/ApiContext.php

```diff
@@ -321,12 +321,21 @@ public function theResultKeyShouldEqualNode($key, $node)
     }
 
     /**
-     * @Then the api error should contain ":text"
+     * @Then the api/API error should contain :text
```

Comment:
I don't see the benefit in that, some people prefer to write "api", others "API". This supports both. You can be consistent using one or the other.